hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusGetTopicInfoHandler.java

```diff
@@ -143,8 +142,7 @@ private Optional<ConsensusTopicInfo> infoForTopic(
             if (!isEmpty(meta.adminKey())) info.adminKey(meta.adminKey());
             if (!isEmpty(meta.submitKey())) info.submitKey(meta.submitKey());
             info.autoRenewPeriod(Duration.newBuilder().seconds(meta.autoRenewPeriod()));
-            if (meta.autoRenewAccountNumber() != 0)
-                info.autoRenewAccount(AccountID.newBuilder().accountNum(meta.autoRenewAccountNumber()));
+            if (meta.autoRenewAccountId().accountNum() != 0) info.autoRenewAccount(meta.autoRenewAccountId());
```

Comment:
meta.hasAutoRenewAccountId() ?

Reply:
ExpiryValidatorImpl.validateAutoRenewAccount() check if (accountID.accountNum() == 0L) {
    There is a comment here:        // 0L is a sentinel number that says to remove the current auto-renew account
I am not sure if 0 mean something special in the code.

Reply:
Updated.